tree/dataframe/inc/ROOT/RNTupleDS.hxx

```diff
@@ -32,10 +32,10 @@
 #include <unordered_map>
 
 namespace ROOT {
-namespace Experimental {
+class RNTuple;
 
+namespace Experimental {
```

Comment:
Why are those still in Experimental?  Should they be in Internal and/or Detail?

Reply:
As discussed, at this point only type names that are part of the on-disk format move out of experimental.